node/send_msg_using_node/index.js

```diff
@@ -0,0 +1,26 @@
+// Include voyage module
+const Vonage = require('@vonage/server-sdk')
+
+const vonage = new Vonage({
+  apiKey: "0ab91e43",
+  apiSecret: "81fPXp0J9HbhSC8w"
```

Comment:
Make sure to use function variables (`req.variables`) for configurable values like these.